.github/workflows/build_sqlite_binaries.yml

```diff
@@ -0,0 +1,107 @@
+name: Build SQLite Binaries
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
```

Comment:
I think we can benefit from cross compilation as have all builds run on `linux-latest`